python/ray/tests/test_reference_counting.py

```diff
@@ -527,6 +527,51 @@ def test_basic_nested_ids(one_worker_100MiB):
     _fill_object_store_and_get(inner_oid_bytes, succeed=False)
 
 
+def _check_actor_states(expected):
+    actors = list(ray.actors().values())
+    for actor in actors:
+        assert actor["State"] == expected
+
+
+def check_actor_states(expected, timeout=10):
```

Comment:
You can just use the `wait_for_condition` helper function.

Reply:
something like
```
def _all_actors_dead():
   return all(actor["State"] == DEAD for actor in ray.actors.values())

wait_for_condition(_all_actors_dead, timeout=..)

Reply:
fixed